packages/charts/test/chart-properties.test.js

```diff
@@ -82,14 +82,14 @@ describe('vaadin-chart properties', () => {
     it('should react to additionalOptions object change', async () => {
       chart.additionalOptions = { title: { text: 'Updated title' } };
       await oneEvent(chart, 'chart-redraw');
-      // await oneEvent(chart, 'chart-redraw');
+      // Await oneEvent(chart, 'chart-redraw');
```

Comment:
nit: this doesn't look right

Reply:
Removed those comments because I don't see them helpful. As for commented-out code in general, you can add `FIXME` or `TODO` one line above a commented-out block to make the rule skip this block.